File: cast-screen.py
import time
import pychromecast

print("Looking for Chromecast devices...")
chromecasts = pychromecast.get_chromecasts()
cast = next(cc for cc in chromecasts if cc.device.friendly_name == "Chromecast-CJ")
cast.wait()
mc = cast.media_controller

url = 'http://192.168.0.10/live/index.m3u8'
content_type = 'application/vnd.apple.mpegurl' 
print("Playing...")
mc.play_media(url, content_type, stream_type="LIVE")
mc.block_until_active()
print(mc.status)
# mc.skip()

File: cast-screen.js
var ChromecastAPI = require('chromecast-api')

var browser = new ChromecastAPI.Browser()

var urlMedia = { url : 'http://192.168.0.10:8090/rpi.webm'
               , contentType: 'video/webm'
               , streamType: 'LIVE'
               }

var checkStatus = function(device) {
    device.getStatus(function(err, status) { console.log(status) })
    setTimeout(function() { checkStatus(device); }, 1000)
}
var update = function(device) {
    device.play(urlMedia, 0, function () {
        console.log('Playing in your chromecast')

        checkStatus(device)
    })
}

browser.on('deviceOn', function (device) {
   update(device)
})
